lib/include.js

```diff
@@ -903,7 +903,7 @@ Inclusion.include = function(objects, include, options, cb) {
           }
         }
         const targetId = obj[relation.keyFrom];
-        if (targetId) {
+        if (targetId != null) {
```

Comment:
Could you please use ``!==`` as such. The !== has faster performance, than the != operator because the Strict Equality Comparison Algorithm doesn't typecast values.

```ts
if (targetId !== null) 

```